src/parse/index.ts

```diff
@@ -142,13 +152,9 @@ export const parse = (
   }
 
   /**
-   * Match `re` and return captures.
+   * Update position and css string. Return the matches
    */
-  function match(re: RegExp) {
-    const m = re.exec(css);
-    if (!m) {
-      return;
-    }
+  function processMatch(m: RegExpExecArray) {
     const str = m[0];
     updatePosition(str);
     css = css.slice(str.length);
```

Comment:
This is the bulk of the changes. The rest is just adaptation for this internal API change.